x-pack/plugins/security_solution/public/management/components/artifact_list_page/components/artifact_flyout.tsx

```diff
@@ -249,8 +261,28 @@ export const MaybeArtifactFlyout = memo<ArtifactFlyoutProps>(
         // Close the flyout
         // `undefined` will cause params to be dropped from url
         setUrlParams({ id: undefined, show: undefined }, true);
+
+        clearItem();
+        onSuccess();
```

Comment:
good catch on  adding `onSuccess` here.

I also don't think you need the `clearItem()` here because the flyout wil be closed out (unmounted)

Reply:
same as above comment, item was persisting on flyout close but if it's resolved, I can remove this.